.github/workflows/ci-linux.yml

```diff
@@ -0,0 +1,231 @@
+        make -j 4
+        sudo make install-strip
+        sudo ldconfig
+
+    - name: Save Hamlib cache
+      if: steps.cache-hamlib.outputs.cache-hit != 'true'
+      uses: actions/cache/save@v4
+      with:
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-hamlib
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/hamlib
+          
+ ## fftw
+    - name: Cache Check fftw 3.3.10
+      id: cache-fftw
+      uses: actions/cache@v4
+      with:
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/fftw
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-fftw
+
+    - name: Install fftw 3.3.10
+      if: steps.cache-fftw.outputs.cache-hit != 'true'
+      id: install-fftw
+      run: |
+        curl --location --output ./fftw-3.3.10.tar.gz https://www.fftw.org/fftw-3.3.10.tar.gz
+        tar zxf fftw-3.3.10.tar.gz
+        cd fftw-3.3.10
+        ./configure --prefix=/home/runner/work/JS8Call-improved/JS8Call-improved/local/fftw --enable-threads --enable-shared --enable-sse2 --enable-avx --enable-avx2 --enable-float
+        make
+        sudo make install
+
+    - name: Save fftw 3.3.10 cache
+      if: steps.cache-fftw.outputs.cache-hit != 'true'
+      uses: actions/cache/save@v4
+      with:
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-fftw
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/fftw
+        
+## boost
+    - name: Cache Check Boost 1.88
+      id: cache-boost
+      uses: actions/cache@v4
+      with:
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/boost
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-boost
+
+    - name: Install Boost 1.88
+      if: steps.cache-boost.outputs.cache-hit != 'true'
+      id: install-boost
+      run: |
+        curl --location --output ./boost-1.88.0.tar.gz https://archives.boost.io/release/1.88.0/source/boost_1_88_0.tar.gz
+        tar zxf boost-1.88.0.tar.gz
+        cd boost_1_88_0
+        ./bootstrap.sh --prefix=/home/runner/work/JS8Call-improved/JS8Call-improved/local/boost
+        sudo ./b2 install
+
+    - name: Save Boost 1.88 cache
+      if: steps.cache-boost.outputs.cache-hit != 'true'
+      uses: actions/cache/save@v4
+      with:
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-boost
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/boost
+        
+## QT to change between 6.8.3 and 6.9.3 just change the 8 to a 9 everywhere here and at the bottom
+    - name: Cache Check QT 6.9.3
+      id: cache-qt
+      uses: actions/cache@v4
+      with:
+        path: /home/runner/work/JS8Call-improved/JS8Call-improved/local/qt693-install
+        key: ${{ runner.os }}-${{ steps.which-os.outputs.thisos }}-${{ runner.arch }}-qt693
```

Comment:
I like it that you embed the version number 693 into the cache key for qt.

Would that not be beneficial to also do for the other libraries (hamlib, fftw, ...)? We'll use different versions of those some time in the future.